package.json

```diff
@@ -64,7 +64,6 @@
     "dompurify": "3.0.1",
     "electron-updater": "5.3.0",
     "express": "4.18.1",
-    "ffi-napi": "4.0.3",
```

Comment:
Will we need to update the snippet about `ffi-napi` in our README with this removal?

Reply:
Likely; I opted not to touch it yet because we may still need those deps (I see that we still end up with `node-gyp-build` because of `native-reg`, `bufferutil`, and `utf-8-validate`).  I was hoping to look at it again after this.